indico/modules/events/papers/client/js/index.js

```diff
@@ -80,6 +80,47 @@ import setupReactPaperTimeline from './setup';
       applySearchFilters();
     }
   };
+
+  global.setupConflictsList = function setupConflictsList() {
+    const $roleLabels = $('.js-assign-dialog .name-column .affiliation > span');
+    $roleLabels.qbubble({
+      show: {
+        event: 'mouseover',
+      },
+      hide: {
+        fixed: true,
+        delay: 100,
+        event: 'mouseleave',
+      },
+      position: {
+        my: 'left center',
+        at: 'right center',
+      },
+      content: {
+        text() {
+          const $this = $(this);
+          const html = $('<div>');
+          const role = $('<strong>', {text: '{0}'.format($(this).data('role-name'))});
```

Comment:
`'{0}'.format(x)` and `x` are basically the same (assuming it's always a string, which seems likely)

I'd fix it in the code you copied this from as well...

Reply:
okay :sweat_smile: